### ISSUE TITLE
Highlighting Values with Negative Numbers

### ISSUE THREAD

**Issue by blentz100** [RobinHerbots/jquery.inputmask#2714]
First off thanks for the great plugin and all the work that goes into it. 

My scenario/question:

There is a field with an existing negative number (for example -3). A user highlights that field and input3 the number 4. In my codepen setup the resulting output in the field is -4. Can anyone recommend the best way to make that behavior result in  an output that show just 4?

In other words, if a user highlights the whole field with a negative sign, and types over it, I'd like that negative sign to disappear. In my codepen I have two different fields that do this, one is a numeric and one is a decimal. They both behave the same way in this codepen, but I'd like to change them if possible to lose the minus sign when a user highlights and types over the field. 

- Add a link to a codepen, jsfiddle or other example page which shows the problem
https://codepen.io/BrendanLentz/pen/dygrVBj?editors=1010
- OS: MacOS Ventura 13.3.1 (a)
- Browser: Brave  1.50.114
- Inputmask version: 5.0.7

**Comment by RobinHerbots** [comment]
@blentz100 ,

Have you tried with version 5.0.8?

**Comment by blentz100** [comment]
@RobinHerbots 

Thank you for your quick reply. 

Yes I have tried with 5.0.8. I just switched the codepen above to using 5.0.8. When I use 5.0.8, it fixes the problem for the second field in the codepen, the one with this input mask:

```
$("#two").inputmask(
  'decimal',
  {
    max: 5,
    min: -5,
    digits:1,
    radixPoint: '.',
    groupSeparator: ',',
  });
```

but it doesn't have any effect on the first field, the one with this input mask:

```
$("#one").inputmask(
  'numeric',
  {
    digits:0,
    groupSeparator: ','
  });
```

So, in other words, in version 5.0.8, field #two works but field #one doesn't. Whereas in version 5.0.7 they both don't work. Any thoughts on that?
